src/banana/data/genpdf/export.py

```diff
@@ -0,0 +1,131 @@
+import io
+import pathlib
+import re
+
+import yaml
+
+
+def list_to_str(ls, fmt="%.6e"):
+    """
+    Convert a list of numbers to a string
+
+    Parameters
+    ----------
+        ls : list(float)
+            list
+        fmt : str
+            format string
+
+    Returns
+    -------
+        str :
+            final string
+    """
+    return " ".join([fmt % x for x in ls])
+
+
+def array_to_str(ar):
+    """
+    Convert an array of numbers to a string
+
+    Parameters
+    ----------
+        ar : list(list(float))
+            array
+
+    Returns
+    -------
+        str :
+            final string
+    """
+    table = ""
+    for line in ar:
+        table += ("% .8e " % line[0]) + list_to_str(line[1:], fmt="%.8e") + "\n"
```

Comment:
why line[0] is separated?

Reply:
I don't know if there is a reason, it's not mine. Maybe @felixhekhorn or @AleCandido can answer. Than, If necessary, I can fix it :)

Reply:
This is because (some?) PDF sets from the website are formatted that way and we retained that behaviour - do we want to change?

Reply:
The difference is that `"% .8e "` has a space after the `%`, in this way is always keeping one character for the sign, that is a minus character if negative, and a space if positive.

Most likely, there would be issues with lhapdf reader without that space...

Reply:
I see, okay we can keep it, for safety